webclient/http2/src/test/resources/certificate_notes.md

```diff
@@ -0,0 +1,3 @@
+To generate a new certificate:
+
+Refer to tests/integration/webserver/upgrade/src/main/resources/certificate_notes.md
```

Comment:
Can you create a `README.md` file under webclient/http2 instead.

Also, I'd say we should copy the instructions instead of referring to another file.

Reply:
Done both